src/renderer/components/+preferences/add-helm-repo-dialog.scss

```diff
@@ -0,0 +1,24 @@
+.AddHelmRepoDialog {
+  --flex-gap: #{$margin * 1.5};
+
+  .accordion {
+    font-weight: bold;
+    text-align: left;
+    padding: 0;
+    color: inherit;
+    &:focus:not(:active) {
+      color: inherit;
+      box-shadow: 0 0 0 0px inset;
+    }
+  }
```

Comment:
Replace `.accordion` class with following statements
```
.accordion {
    font-weight: bold;
    align-self: flex-start;
    padding: 0;

    &:focus:not(:active) {
      box-shadow: none;
    }
  }
```
We want to preserve focus state on element.

Reply:
Fixed. As for me, looks a little bit strange :)
![image](https://user-images.githubusercontent.com/53330707/101157419-86f76100-3643-11eb-9c3a-50fe95ce2571.png)

Reply:
Oh, sorry, probably we need to restore `color: inherit` into parent `accordion` class. The intention is to keep visible focused state, so button should be blue with focus, but gray without focus.

Reply:
It works how you said, just I would like to leave it grey

Reply:
Hm, I was not understanding what is strange in this accordion, now I got it, configuration-secrets-create form contains similar things, but they are not "focusable" (I stole the idea from that form)